widgets/hits-per-page-selector/hits-per-page-selector.js

```diff
@@ -41,14 +41,14 @@ function hitsPerPageSelector({
   return {
     init: function(state) {
       let isCurrentInOptions = reduce(options, function(res, option) {
-        if (state.hitsPerPage === undefined) {
-          return true;
-        }
         return res || +state.hitsPerPage === +option.value;
       }, false);
 
       if (!isCurrentInOptions) {
-        throw new Error('[hitsPerPageSelector]: No option in `options` with `value: ' + state.hitsPerPage + '`');
+        options = [{value: undefined, label: ''}].concat(options);
+        if (console) {
```

Comment:
You should check window.console (we are only browser based right now). Old IES have no global console and the check will throw I think.

And use console.log (olds IEs have no console.warn).

Sadly polyfill.io that we advise to use has now console polyfill, we might add one as a PR to them at some point.

Also I think at some point we will only console.* only for DEV purposes (so when NODE_ENV is set and equals to development). For now it's ok